lib/cloudregister/smt.py

```diff
@@ -40,6 +40,11 @@ def __init__(self, smtXMLNode, https_only=False):
         except KeyError:
             self._region = 'unknown'
         self._fqdn = smtXMLNode.attrib['SMTserverName']
+        try:
+            self._registry_fqdn = 'registry-' + self._fqdn.split('-')[1]
```

Comment:
I think we need to talk about extending the region server to send this FQDN, instead of inferring it.